packages/ods-react/src/components/tag/src/dev.module.css

```diff
@@ -0,0 +1,5 @@
+.ods-tag {
```

Comment:
you may also need to update the `module.d.ts` file with `declare module '*.css';` to make TS happy